wcc-contentful-app/lib/wcc/contentful/app.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'wcc/contentful/rails'
```

Comment:
has this file moved to the app gem?

Reply:
no, but if you require `wcc-contentful-app` then `wcc/contentful/rails` is a dependency.  In the past the Gemfile was set up like so:

```
gem 'wcc-contentful', require: 'wcc/contentful/rails'
```

Now, we shouldn't even need `gem 'wcc-contentful'` because `gem 'wcc-contentful-app'` includes it as a dependency.  Therefore `wcc-contentful-app` needs to require the rails section.